TEST_DEV_GUIDE.md

```diff
@@ -103,15 +102,15 @@ Defining test steps require using and extending the commands and operations to b
 
 ## Implementing Test Steps
 
-Follow standard best practices for implementing test steps in Behave, writing the test steps as if the feature is fully supported and then adding code at all levels to support the new test. The process is something like this:
+Follow standard best practices for implementing test steps in Behave, writing the test steps as if the feature is fully supported and then adding code at all levels to supprt the new test. The process is something like this:
 
 - Study the RFC and define the tests needed to cover at least the happy path requirements of the RFC
   - Raise as issues in the [Aries RFCs](https://github.com/hyperledger/aries-rfcs) repo any deficiencies in the RFC
 - Define Gherkin tests using the existing persona
   - Add new persona only if really, really needed
 - Execute the new tests, grabbing the skeleton code for the new, unimplemented steps from the behave output, and adding them to the `steps` Python code
 - Implement the steps code, defining as needed commands and operations to be added to the backchannel interface
-- Update the Google Sheet and OpenAPI spec to add the new commands and operations and regenerate the backchannel operations data file (see [subsection](#defining-backchannel-operations) above)
+- Update the Google Sheet to add the new commands and operations and regenerate the backchannel operations data file (see [subsection](#defining-backchannel-operations) above)
```

Comment:
This seems wrong -- looks like things are being undone?  We need to update the OpenAPI Spec.

Perhaps a restore bug?

Reply:
File properly merged in new commit.